types/types.go

```diff
@@ -513,6 +513,8 @@ type SystemContext struct {
 	// === dir.Transport overrides ===
 	// DirForceCompress compresses the image layers if set to true
 	DirForceCompress bool
+	// IsCachedImage indicate if the image being pulled is already cached
+	IsCachedImage bool
```

Comment:
This is kind of interesting... it's really specific to an instance of `systemContext` use... i'm not sure about this - especially if systemContext is used for multiple things, if this would make sense.

Reply:
Would it work to put it in `BlobInfo` to pass it down?

Reply:
I have to pass the boolean to indicate if it's a cached image from here, https://github.com/harche/cri-o/blob/with_enclib/server/image_pull.go#L92 

The only thing that gets passed down from there is systemContext. This is similar to how `ImagePullSecrets` as well as our `ImageDecryptSectes` are passed on.